src/components/split-button/split-button.style.ts

```diff
@@ -5,18 +5,48 @@ import baseTheme from "../../style/themes/base";
 
 const StyledSplitButton = styled.div`
   ${margin}
-
   display: inline-block;
   position: relative;
 
+  ${StyledButton}:first-of-type {
+    border-bottom-right-radius: var(--borderRadius000);
+    border-top-right-radius: var(--borderRadius000);
+    border-bottom-left-radius: var(--borderRadius400);
+    border-top-left-radius: var(--borderRadius400);
+  }
+
+  ${StyledButton} {
+    &:hover {
+      border-bottom-right-radius: var(--borderRadius000);
+      border-top-right-radius: var(--borderRadius000);
+      border-bottom-left-radius: var(--borderRadius400);
+      border-top-left-radius: var(--borderRadius400);
+    }
+  }
+
+  a:hover {
+    border-radius: 0px;
+  }
+
   & > ${StyledButton} {
     margin: 0;
-
     &:focus {
       border: 3px solid var(--colorsSemanticFocus500);
       outline: none;
+      border-bottom-right-radius: var(--borderRadius400);
```

Comment:
Same with this too - As you've got this block of styling twice, it might be best to have this declared as a const and refer to it in the blocks of css. Just means if we need to change the tokens in future, we do it in one place instead of two.